client/my-sites/upgrades/domain-management/components/domain/maintenance-card.jsx

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import React from 'react' ;
+
+/**
+ * Internal dependencies
+ */
+import { localize } from 'i18n-calypso';
+import { getSelectedDomain } from 'lib/domains';
+import Notice from 'components/notice';
+
+const MaintenanceCard = ( { domains, selectedDomainName, translate } ) => {
+	const domain = getSelectedDomain( { domains, selectedDomainName } ),
+		lastIndexOfDot = domain.name.lastIndexOf( '.' ),
+		tld = lastIndexOfDot !== -1 && domain.name.substring( lastIndexOfDot );
```

Comment:
Wanted to do in another PR, as it's pretty out of scope, and testing gets a bit biggish, but will do. ;)